sdk/python/kfp/__init__.py

```diff
@@ -16,13 +16,12 @@
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)
 
-__version__ = '1.8.11'
+__version__ = '2.0.0rc0'
```

Comment:
rc should be very close to a stable release. Our first release would be a beta release. And for this change specifically, we don't plan for cutting any public release, so I'd probably put alpha here. WDYT?